sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -435,6 +467,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-tgl-rt715-rt711-rt1308-mono.tplg",
 	},
+	{
+		.link_mask = 0xF, /* 4 active links required */
+		.links = tgl_712_only,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-tgl-rt712.tplg",
+	},
```

Comment:
should we add this for ADL/RPL/MTL as well?

Reply:
> should we add this for ADL/RPL/MTL as well?

That is an open. The target is MTL, but currently, it is only tested on TGL Up Xtreme board. I don't know is ADL/RPL needed.